views/js/qtiCreator/editor/ckEditor/htmlEditor.js

```diff
@@ -215,6 +215,19 @@ define([
                     if ($toolbarArea) {
                         $toolbarArea.hide();
                     }
+
+                    let html = this.getData($editable);
+                    const tagsToMerge = [
+                        'strong',
+                        'em',
+                        'sub',
+                        'sup'
+                    ];
+                    tagsToMerge.forEach(function(tag) {
+                        let regex = new RegExp("<\\/"+tag+">(\\s*)<"+tag+">", 'gi');
+                        html = html.replace(regex, "$1");
+                    });
+                    $editable.data('editor').setData(html);
```

Comment:
I found better place for this https://github.com/oat-sa/extension-tao-itemqti/blob/e9f3adf4ed2dc50e7e5536e6300364967f61b309/views/js/qtiXmlRenderer/renderers/Container.js#L37-L58
It is xmlRenderer that will be used on Save